sys-fs/btrfs-progs/btrfs-progs-4.3.1.ebuild

```diff
@@ -72,10 +82,36 @@ src_configure() {
 }
 
 src_compile() {
-	emake V=1
+	local static=""
+
+	if use static || use static-libs; then
+		static="static"
+	fi
+
+	emake V=1 $static
```

Comment:
This `$static` varaible could be replaced with a usev call.

Also, I think we also want to build/install the normal dynamically linked binaries since the static binaries have funny names. That means we have to explicitly name the `all` target.

```
emake V=1 all $(usev static)
```